### DISCUSSION TITLE
헷갈리는 기획 부분에 대해 정리해보아요

### DISCUSSION THREAD

**Discussion by woo-chang** [woowacourse-teams/2023-votogether#265] [기획]
API를 하나씩 살펴보면서 헷갈리는 부분이 존재했습니다! (얘기는 나눴지만, 제 기억력이 안 좋아서 기억을 못 하는 걸 수도 있는 것 같아요 🥲)
기획적으로 헷갈리는 부분에 대해 아래와 같이 정리해 보았고, 같이 얘기해보면 좋을 것 같아요 :)

## 회원

- 포인트 관련 기능은 현재 없는데, 포인트를 삭제해도 괜찮을까요?

## 게시글

- 게시글 작성 시 본문에 최대 몇 장의 사진을 넣을 수 있나요?
- 게시글에 사진 등록 시 어떻게 보여지나요?
- 게시글 목록 조회 시 페이징 처리는 몇 개가 좋을까요?
- 게시글 조기 마감은 조건 없이 언제든 가능한가요?
- 게시글 수정 시 마감 시간 수정이 가능한가요?
- 초기 카테고리에는 어떤 것들이 들어가면 좋을까요?

## 신고

- 게시글, 댓글, 닉네임은 각각 몇 회 이상 신고되어야 제재가 들어가나요?
- 게시글, 댓글, 닉네임은 각각 어떤 제재가 들어가나요?

**Comment by chsua** [comment]
마감이 되지 않은 글도 삭제처리 할 수 있나요?

**Comment by aiaiaiai1** [comment]
로그인할때 성별이랑 나이 입력하는 페이지는 어떻게 되는건지?

**Comment by Gilpop8663** [comment]
카카오톡 로그인 시 성별, 나이대가 설정되지 않으면 오류가 나고 있습니다 유저가 설정하지 않았다면 프론트에서 회원의 유저의 정보를 받고 백엔드로 정보를 보내줘서 회원 생성을 하나요?

**Comment by Gilpop8663** [comment]
게시글 수정 시 투표가 되었다면 수정이 되지 않는 것 유효하죠?

**Comment by jeomxon** [comment]
## 회원

`Q` : 포인트 관련 기능은 현재 없는데, 포인트를 삭제해도 괜찮을까요? 

`A` : 포인트는 삭제합니다.

<br>

`Q` : 로그인할때 성별이랑 나이 입력하는 페이지는 어떻게 되는건지?

`A` : 로그인(회원가입) 후에 성별, 연령대 등을 입력 무조건 하도록 redirect합니다.

   카카오 선택동의는 제거하고 필수동의(이메일)만 받습니다.

<br>
<hr>

## 게시글

`Q` : 게시글 작성 시 본문에 최대 몇 장의 사진을 넣을 수 있나요?

`A` : `1`장 넣어줄 수 있습니다.

<br>

`Q` : 게시글에 사진 등록 시 어떻게 보여지나요?

`A` : `1`장만 하단에 보여집니다.

<br>

`Q` : 게시글 목록 조회 시 페이징 처리는 몇 개가 좋을까요?

`A` : `10`개로 진행합니다.

<br>

`Q` : 게시글 조기 마감은 조건 없이 언제든 가능한가요?

`A` : 조기마감은 조건 없이 가능합니다.

<br>

`Q` : 게시글 수정 시 마감 시간 수정이 가능한가요?

`A` : 마감시간 수정도 가능합니다. 하지만 현재 시간 이전일 수는 없습니다.

<br>

`Q` : 게시글 삭제는 조건 없이 가능한가요?

`A` : `20`명 이상 투표한 경우 삭제할 수 없습니다.

<br>

`Q` : 초기 카테고리에는 어떤 것들이 들어가면 좋을까요?

`A` : IT, 연애, 음식, 운동, 재테크, 음악, 동물, 고민, 잡담

<br>

`Q` : 게시글 수정 시 투표가 되었다면 수정이 되지 않는 것 유효하죠?

`A` : 네 유효합니다.

<br>
<hr>

## 신고

`Q` : 게시글, 댓글, 닉네임은 각각 몇 회 이상 신고되어야 제재가 들어가나요?

`A` : 5, 5, 3회 이상 신고가 되어야 제재됩니다.

<br>

`Q` : 게시글, 댓글, 닉네임은 각각 어떤 제재가 들어가나요?

`A` : 게시글, 댓글 블라인드 처리, 닉네임은 임의로 변경 합니다.

<br>

게시글, 댓글에 대한 블라인드처리는 조회 시 아예 보여주지 않는걸로 한다!

**Comment by woo-chang** [comment]
## 게시글

`Q` : 투표가 존재하면 게시글 수정이 가능한가요?

`A` : 투표가 존재하면 게시글 수정이 불가능합니다.

`Q` : 마감되지 않은 게시글도 삭제가 가능한가요?

`A` : 마감된 게시글만 삭제가 가능합니다.

**Comment by woo-chang** [comment]
## 회원

`Q` : 성별, 나이대는 한번 입력 후 수정이 불가능한가요?

`A` : 조작이 너무 쉬워질 것 같아서 초기에 한번만 입력하도록 합니다.

**Comment by woo-chang** [comment]
`Q` : 투표 개수가 몇 개 이상부터 게시글을 삭제할 수 없나요?

`A` : 투표 개수가 20개 이상이면 게시글을 삭제할 수 없습니다.

**Comment by woo-chang** [comment]
`Q` : 게시글 본문에 사진 `1`장만 넣는 것이 유효한가요?

`A` : 현재는 `1`장까지만 유효합니다.

`Q` : 게시글 투표 최대 `3`일까지 가능하다는 것 유효한가요?

`A` : 게시글 투표 최대 `14`일까지로 변경되었습니다.

`Q` : 검색 시에도 카테고리 필터링이 적용되는건가요?

`A` : 검색 시에는 카테고리 필터링이 적용되지 않습니다.

**Comment by woo-chang** [comment]
`Q` : 게시글 제목 글자 수는 몇 자까지 가능한가요?

`A` : 최대 100자까지 입력 가능합니다.

`Q` : 게시글 내용 글자 수는 몇 자까지 가능한가요?

`A` : 최대 1000자까지 입력 가능합니다.
